src/Admin/OnboardingWizardSubmenuPage.php

```diff
@@ -254,6 +263,7 @@ public function enqueue_assets( $hook_suffix ) {
 			'UPDATES_NONCE'                      => wp_create_nonce( 'updates' ),
 			'USER_FIELD_DEVELOPER_TOOLS_ENABLED' => UserAccess::USER_FIELD_DEVELOPER_TOOLS_ENABLED,
 			'USERS_RESOURCE_REST_PATH'           => '/wp/v2/users',
+			'AMP_THEMES'                         => wp_list_pluck( AMPThemes::$themes, 'slug' ),
```

Comment:
The `AMPThemes` service should rather be added as a dependency for this class, and then instead of calling `set_themes` above, this line could do something like:

```suggestion
			'AMP_THEMES'                         => wp_list_pluck( $this->ampThemes->get_themes(), 'slug' ),
```

As noted about plugins, the `get_themes` method can load the data from the filesystem the first time it is needed (or it could also just do `return include AMP__DIR__ . '/includes/amp-themes.php';` as noted above.